crates/sui/src/client_commands.rs

```diff
@@ -174,6 +174,27 @@ pub enum SuiClientCommands {
         address_override: Option<ObjectID>,
     },
 
+    /// Run the bytecode verifier on given package and report statistics.
+    #[clap(name = "bytecode-verify")]
+    BytecodeVerify {
```

Comment:
This command name could be better

Reply:
also wondering if this could be added to `VerifySource`?

Reply:
Do we expect other people (outside us) to use this command?  If not, we should consider hiding it:

```
    #[clap(name = "Xbytecode-verify", hidden = true)]
    BytecodeVerify {
```